### PR TITLE
Create send max button and fee types

### PR DESCRIPTION
**Pull request by dangershony**

### CODE REVIEW

File: src/Networks/Xds/Blockcore.Networks.Xds/XdsMain.cs [hunk 1]
```diff
@@ -43,10 +43,10 @@ public XdsMain()
             this.MaxTimeOffsetSeconds = 25 * 60;
             this.DefaultBanTimeSeconds = 8000;
             this.MaxTipAge = 2 * 60 * 60;
-            this.MinTxFee = Money.Coins(0.00001m).Satoshi;
+            this.MinTxFee = Money.Coins(0.01m).Satoshi;
```
Comment (aubergemediale) [inline src/Networks/Xds/Blockcore.Networks.Xds/XdsMain.cs:46]: hard fork

File: src/Networks/Xds/Blockcore.Networks.Xds/XdsMain.cs [hunk 2]
```diff
@@ -43,10 +43,10 @@ public XdsMain()
...
             this.MaxTxFee = Money.Coins(1).Satoshi;
```
Comment (aubergemediale) [inline src/Networks/Xds/Blockcore.Networks.Xds/XdsMain.cs:47]: hard fork

**Comment by thecrypt0hunter** [comment]
@sondreb starting to make some changes to the home page based on this issue https://github.com/block-core/blockcore/issues/147

File: src/Networks/Xds/Blockcore.Networks.Xds/XdsMain.cs (hunk 2 above)
Comment (dangershony) [reply src/Networks/Xds/Blockcore.Networks.Xds/XdsMain.cs:47]: Why? This is set to be the same as absolutely min fee.
Besides blocks won't get rejected based on fee it's just mempool relay problems at worth case will get effected

**Pull request merged**